### PR TITLE
Add MDL and CDL

### PR DESCRIPTION
**Pull request by anarchivist**

### CODE REVIEW

File: lib/fetchers/mdl_api_fetcher.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        content = parsed_content.get("response")
+
+        if not self.total_records:
+            total_records_prop = "numFound"
+            self.total_records = getprop(content, total_records_prop)
+
+        if content is None:
+            error = "Error, there is no \"response\" field in content from " \
+                    "URL %s" % url
+        # elif exists(content, "response/headers/code"):
```
Comment (markbreedlove) [inline lib/fetchers/mdl_api_fetcher.py]: Should this be commented out?

If this action should not be taken, would it be more clear with a comment above saying what's not going to be done?

File: lib/akamod/cdl_identify_object.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    """
+
+    try:
+        data = json.loads(body)
+    except:
+        response.code = 500
+        response.add_header('content-type', 'text/plain')
+        return "Unable to parse body as JSON"
+
+    #handle_field = "originalRecord/handle"
```
Comment (markbreedlove) [inline lib/akamod/cdl_identify_object.py]: It looks like this line can probably be deleted.

File: .gitignore [hunk 3]
```diff
@@ -10,4 +10,5 @@ caches/*
 akara.ini
 akara.conf
 akara.confc
-ENV/*
\ No newline at end of file
+ENV/*
+couchdb.*
```
Comment (markbreedlove) [inline .gitignore]: What are these `couchdb.*` files for?

Comment (anarchivist) [inline .gitignore]: IIRC CouchDB logs are generated in the current working directory when you start it as a background process with `couchdb -b`.

**Comment by anarchivist** [comment]
@markbreedlove The commented out code you identified has been now removed, and this branch has been rebased against the latest `develop`.

**Comment by markbreedlove** [comment]
OK, I don't have any objection to its being merged, then.  Would you like to squash it?

**Comment by anarchivist** [comment]
Given the complexity of the CDL work, I'd like to retain the commit history if that's OK with you.

**Comment by markbreedlove** [comment]
OK. That shouldn't be a problem.

**Pull request merged**